src/app/components/contact-list/contact-list.component.ts

```diff
@@ -85,23 +80,29 @@ export class ContactListComponent implements OnInit, OnDestroy {
             });
     }
 
+    ngAfterContentInit() {
+        this.calculateItemsPerRow();
+    }
+
+    @HostListener('window:resize', ['$event'])
+    onResize() {
+        this.calculateItemsPerRow();
```

Comment:
Does this work with a lot of contacts? If it causes lagging then a debounce might help

Reply:
It wasn't heavily lagging but I only tested it 5 contacts. A debounce makes sense for sure. Thank you!